src/main/java/org/folio/rest/impl/AutomatedPatronBlocksAPI.java

```diff
@@ -50,18 +60,28 @@ public void postAutomatedPatronBlocksSynchronizationJob(SynchronizationJob reque
     Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
     Context vertxContext) {
 
+    log.debug("getAutomatedPatronBlocksByUserId:: parameters request: {}, okapiHeaders: {}",
+      () -> logAsJson(request), () -> logOkapiHeaders(okapiHeaders));
+
+    Handler<AsyncResult<Response>> loggingResponseHandler =
+      loggingResponseHandler("postAutomatedPatronBlocksSynchronizationJob", asyncResultHandler, log);
+
     new SynchronizationJobService(okapiHeaders, vertxContext.owner())
       .createSynchronizationJob(request)
-      .onSuccess(response -> asyncResultHandler.handle(succeededFuture(
+      .onSuccess(response -> loggingResponseHandler.handle(succeededFuture(
         PostAutomatedPatronBlocksSynchronizationJobResponse
           .respond201WithApplicationJson(response))))
       .onFailure(throwable -> {
         String errorMessage = throwable.getLocalizedMessage();
+        log.warn("getAutomatedPatronBlocksByUserId:: Failed to create synchronization job",
+          throwable);
         if (throwable instanceof UserIdNotFoundException) {
+          log.debug("getAutomatedPatronBlocksByUserId:: User ID not found");
           asyncResultHandler.handle(succeededFuture(
             PostAutomatedPatronBlocksSynchronizationJobResponse.respond422WithTextPlain(
               errorMessage)));
         } else {
+          log.debug("getAutomatedPatronBlocksByUserId:: unexpected error");
```

Comment:
worn level?

Reply:
The error is logged above with a stack trace, here we're just trying to log the type of the error, which BTW may be redundant. Actually, I think I'm going to remove these logs because the type will be logged by the log.warn() above.